doc/extensibility/IDynamicEnumValuesProvider.md

```diff
@@ -33,8 +33,9 @@ Provides a dynamic list of enum values using an `IDynamicEnumValuesGenerator`. Y
     }
 ```
 ##IDynamicEnumValuesGenerator
-**Note:** Issue [#53](https://github.com/Microsoft/VSProjectSystem/issues/53)
+~~**Note:** Issue [#53](https://github.com/Microsoft/VSProjectSystem/issues/53)~~
```

Comment:
I don't think you have to create this link yourself. Merely using "#53" should be enough to create the link. Did you find it otherwise?